### PR TITLE
Refactor nuts-node chart for separate internal and external services

### PR DESCRIPTION
**Pull request by rolandgroen**
This revision refactors nuts-node charts to support separate internal and external services. It provides different ingress classes, annotations, hosts, and rules for external and internal services. It also adjusts `service.yaml` to accommodate these changes and provides different application URLs for internal and external services. Configurations in `values.yaml` have been altered to reflect these changes as well.

### CODE REVIEW

**Review by JorisHeadease — APPROVED** [review]
Kleine comment toegevoegd, niet een heel belangrijke. Ik approve de PR alvast aangezien dit geen functionele impact heeft

**Comment by rolandgroen** [comment]
@gerardsn could you pls review this before we merge.

File: charts/nuts-node/values.yaml [hunk 1]
```diff
@@ -99,8 +120,11 @@ nuts:
...
     datadir: /opt/nuts/data
     http:
+      internal:
+        address: :8081
+      external:
+        address: :8080
       default:
-        address: :1323
         cors:
           origin: "https://chart-example.local"
```
Comment (gerardsn) [inline charts/nuts-node/values.yaml:129]: the http.default section no longer exists

**Review by gerardsn — APPROVED** [review]
I'm no Kubernetes expert, so just skimmed through the PR. If tested and works I'm fine merging this

File: charts/nuts-node/values.yaml (hunk 1 above)
Comment (rolandgroen) [reply charts/nuts-node/values.yaml:129]: In the files README.rst komt nog de tekst voor:

> As a general safety precaution ``auth.contractvalidators`` ignores the ``dummy`` option if configured,
requesting an access token from another node on ``/n2n/auth/v1/accesstoken`` does not return any error details,
``auth.accesstokenlifespan`` is always 60 seconds, ``http.default.cors.origin`` does not allow a wildcard (``*``),
json-ld context can only be downloaded from trusted domains configured in ``jsonld.contexts.remoteallowlist``,
and the ``internalratelimiter`` is always on.

Comment (rolandgroen) [reply charts/nuts-node/values.yaml:129]: We halen hem weg uit de config.

**Pull request merged**